### PR TITLE
LCDC names

### PR DESCRIPTION
**Pull request by AlvaroBurnett**

### CODE REVIEW

File: content/LCDC.md [hunk 1]
```diff
@@ -39,13 +39,13 @@ picture sticks to the screen. (TODO: research this more.)
 When re-enabling the LCD, the PPU will immediately start drawing again,
 but the screen will stay blank during the first frame.
 
-## LCDC.6 - Window Tile Map Display Select
+## LCDC.6 - Window tile map range
```
Comment (ISSOtm) [inline content/LCDC.md:42]: Range or area?

**Review by ISSOtm — CHANGES_REQUESTED** [review]

File: content/LCDC.md (hunk 1 above)
Comment (AlvaroBurnett) [reply content/LCDC.md]: Yes, area is better. Fixed

Comment (ISSOtm) [reply content/LCDC.md]: Lines 58 and 67 too, please

Comment (AlvaroBurnett) [reply content/LCDC.md]: Fixed

**Review by ISSOtm — APPROVED** [review]

**Review by avivace — APPROVED** [review]

**Pull request merged**